### DISCUSSION TITLE
always keep tab awake

### DISCUSSION THREAD

**Discussion by lmcr666** [Floorp-Projects/Floorp#1447] [Feature Request]
there are some tabs that i want to keep awaken forever, could you add this feature please?

**Comment by CutterKnife** [comment]
You can always specify a website to keep tab awake.
<img width="686" alt="スクリーンショット 2024-09-23 17 18 00" src="https://github.com/user-attachments/assets/edfea28f-3048-4884-aeb2-72b27f8e8935">

**Reply by lmcr666** [reply]
didn't know this existed. it's exactly what I wanted, thank you so much!
